data/mapcss-mapping.csv

```diff
@@ -1017,7 +1017,7 @@ deprecated|deprecated;901;x
 railway|subway|gray;[railway=subway][colour=gray];x;name;int_name;902;railway|subway|tunnel
 deprecated|deprecated;903;x
 railway|subway|lightblue;[railway=subway][colour=lightblue];x;name;int_name;904;railway|subway|tunnel
-deprecated|deprecated;905;x
+amenity|motorcycle_rental;[amenity=motorcycle_rental],[shop=motorcycle][motorcycle:rental=yes],[shop=motorcycle][rental=yes],[amenity=scooter_rental];;;;905;
```

Comment:
Won't this [shop=motorcycle][motorcycle:rental=yes] overwrite shop=motorcycle?

Reply:
It shouldn't as it'll go into `amenity-motorcycle_rental`.

If the line was `shop|motorcycle|rental;[[shop=motorcycle][motorcycle:rental=yes]` then indeed all motorcycle shops with a rental service would've turned into `shop-motorcycle-rental` which is wrong.

But not the case here.